src/devices/machine/wd2010.cpp

```diff
@@ -703,7 +702,13 @@ void wd2010_device::complete_write_sector(uint8_t data)
 	} // --------------------------------------------------------
 
 	// 'complete_cmd' ON THE FALLING EDGE OF _BUFFER_READY_ ( set by WRITE_SECTOR ) !
-	m_deassert_write_timer->adjust(attotime::from_usec(1), newstatus);
+	if ((m_task_file[TASK_FILE_COMMAND] & 4) && m_task_file[TASK_FILE_SECTOR_COUNT] > 1) {
+		m_task_file[TASK_FILE_SECTOR_COUNT]--;
+		m_task_file[TASK_FILE_SECTOR_NUMBER]++;
+		m_multsect_write_timer->adjust(attotime::from_msec(1), data);
+	} else {
+		m_deassert_write_timer->adjust(attotime::from_usec(1), newstatus);
+	}
```

Comment:
Please use Allman formatting like the existing code in this file.

Reply:
Ok.